City-of-Helsinki/drupal-helfi-etusivu
# UHF-6041: Removed target group links templates and styles

# [UHF-6041](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6041)


## What was done


* Removed target group links styles and templates since they have been moved to hdbt

## How to install

* Make sure your instance is up and running on correct branch.
  * `git pull origin UHF-6041-paragraphs-to-global`
  * `make fresh`
* Run `make drush-cr`

## How to test


See instructions in this PR https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/355

## Designers review
